### PR TITLE
Optimize discover memory usage

### PR DESCRIPTION
**Pull request by brunobar79**
- optimize a few components: CoinIconGroup, CoinRow, UniswapPoolListRow
- add removeClippedSubviews to DiscoverSheet scrollview
- pass dimensions to CDN for optimal assets
- optimize UniswapPoolListRow.js
- refactor useUniswapPools hook
- use no-cache in uniswap queries

Before:

Screen Shot 2021-03-18 at 9 29 11 AM

After: 

 
Screen Shot 2021-03-18 at 6 47 54 PM

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
lgtm 👌

**Review by osdnk — APPROVED** [review]

File: src/components/discover-sheet/UniswapPoolsSection.js [hunk 1]
```diff
@@ -58,6 +61,10 @@ const listData = [
   },
 ];
 
+const renderUniswapPoolListRow = ({ item }) => (
+  <UniswapPoolListRow assetType="uniswap" item={item} />
```
Comment (jinchung) [inline src/components/discover-sheet/UniswapPoolsSection.js:65]: we don't care about key anymore?

File: src/hooks/useUniswapPools.js [hunk 2]
```diff
@@ -146,23 +147,23 @@ async function getBulkPairData(pairList, ethPrice, ethPriceOneMonthAgo) {
           let oneDayHistory = oneDayData?.[pair.id];
           if (!oneDayHistory) {
             const newData = await uniswapClient.query({
-              fetchPolicy: 'network-only',
+              fetchPolicy: 'no-cache',
               query: UNISWAP_PAIR_DATA_QUERY(pair.id, b1),
             });
             oneDayHistory = newData.data.pairs[0];
```
Comment (jinchung) [inline src/hooks/useUniswapPools.js:153]: there was one time when I saw: `error in getBulkPairData [TypeError: undefined is not an object (evaluating 'newData.data.pairs')]` 
haven't been able to reproduce, but we should add some optional chaining

**Comment by jinchung** [comment]
let's prioritize typescripting useUniswapPools after all this :D

**Review by jinchung — APPROVED** [review]
🥑

**Pull request merged**